src/steps/owners/__snapshots__/converters.test.ts.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`#createUserEntity should convert to entity 1`] = `
+Object {
+  "_class": Array [
+    "User",
+  ],
+  "_key": "hubspot_user:84106262 ",
+  "_rawData": Array [
+    Object {
+      "name": "default",
+      "rawData": Object {
+        "archived": false,
+        "createdAt": "2021-06-14T19:38:46.907Z",
+        "email": "user@example.com",
+        "firstName": "John",
+        "id": "84106262",
+        "lastName": "Doe",
+        "updatedAt": "2021-06-14T19:38:46.907Z",
+        "userId": 25150127,
+      },
+    },
+  ],
+  "_type": "hubspot_user",
+  "archived": false,
```

Comment:
Can we also set an `active` property?

> Which users are inactive?

```
find User with active=false
```

Reply:
See parent class of `User`: https://github.com/JupiterOne/data-model/blob/master/src/schemas/Entity.json#L84